File: code1.py
import numpy as np
class Solution(object):
    def twoSum(self, nums, target):
        np_data=np.array(nums)
        for n in np_data:
            if (target-n) in np_data and target-n!=n:
                i,=np.where(np_data==n)
                j,=np.where(np_data==target-n)
                return[i[0],j[0]]
            elif target-n in np_data and target-n==n:
                i,=np.where(np_data==n)
                if len(list(i))>1:
                    return [i[0],i[1]]

File: code1.cpp
#include <bits/stdc++.h>

using namespace std;


class Solution {
public:
    
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> result;
        map<int,int> seen;
        map<int,int> index;
        for(int i=0;i<nums[0];i++){
            if(seen[target-nums[i]]==1){
                result.push_back(index[target-nums[i]]);
                result.push_back(i);
                cout<<i;
                return result;
            }
            seen[nums[i]]=1;
            index[nums[i]]=i;

        }
    }
};

int main(){
    Solution s;
    vector<int> input;
    input.push_back(3);
    input.push_back(3);
    input.push_back(2);
    s.twoSum(input,6);
}